javaslang/src/main/java/javaslang/collection/Arrays2.java

```diff
@@ -63,22 +60,22 @@
         return array;
     }
 
-    static <T> T[] pushFront(T[] array, T element) {
+    static <T> T[] copyPrepend(T[] array, T element) {
```

Comment:
will try to rebase all these modifications to the correct commit, so that I don't modify the same line twice ...